invenio_indexer/config.py

```diff
@@ -22,6 +22,24 @@
 INDEXER_MQ_ROUTING_KEY = "indexer"
 """Default routing key for message queue."""
 
+INDEXER_MQ_PUBLISH_KWARGS = {}
+"""Default message queue producer publishing kwargs.
+
+Passed to ``kombu.Producer:publish``.
+
+.. code-block:: python
+
+    INDEXER_MQ_PUBLISH_KWARGS = {
+        "retry": True,
+        "retry_policy": {         # Setting for maximum waiting time of ~10min:
+            "interval_start": 0,  # First retry immediately,
+            "interval_step": 2,   # then increase by 2s for every retry.
+            "interval_max": 30,   # but don't exceed 30s between retries.
+            "max_retries": 30,    # give up after 30 tries.
+        },
+    }
```

Comment:
Not sure if this should be the default or not... If we did it would be a "breaking" change in case someone relied on falling fast.